orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/config/CloudDriverConfigurationProperties.groovy

```diff
@@ -25,12 +25,18 @@ class CloudDriverConfigurationProperties {
   @Canonical
   static class BaseUrl {
     String baseUrl
+    Map<String, Object> config
+  }
+
+  static class MultiBaseUrl {
+    String baseUrl
+    List<BaseUrl> baseUrls
   }
 
   @Canonical
   static class CloudDriver {
     String baseUrl;
-    BaseUrl readonly;
+    MultiBaseUrl readonly;
```

Comment:
I'll double check again but this is expected to preserve the existing way of configuring a single read-only clouddriver.

```
clouddriver:
  baseUrl: http://something.com
  readonly:
    baseUrl: http://something-readonly.com
```